src/node/src/client.js

```diff
@@ -224,9 +225,11 @@ function _read(size) {
     } catch (e) {
       self._readsDone({code: grpc.status.INTERNAL,
                        details: 'Failed to parse server response'});
+      return;
```

Comment:
Returning here just makes sure that we don't fall through to `self.push(deserialized)`, which we don't want because this is the case where deserialization failed.